docs/additional-features/chainbridge/definitions.md

```diff
@@ -0,0 +1,188 @@
+```
+
+### Pause/Unpause
+
+Pause deposits, proposal creation, voting, and deposit executions temporally.
+
+```bash
+# Pause
+$ cb-sol-cli admin pause \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]"
+
+# Unpause
+$ cb-sol-cli admin unpause \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]"
+```
+
+### Change Fee
+
+Change the fee which will be paid to Bridge Contract
+
+```bash
+# Change fee for execution
+$ cb-sol-cli admin set-fee \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --fee [FEE_IN_WEI]
+```
+
+### Add/Remove a relayer
+
+Add an account as a new relayer or remove an account from relayers
+
+```bash
+# Add relayer
+$ cb-sol-cli admin add-relayer \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --relayer "[NEW_RELAYER_ADDRESS]"
+  
+# Remove relayer
+$ cb-sol-cli admin remove-relayer \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --relayer "[RELAYER_ADDRESS]"
+```
+
+### Change relayer threshold
+
+Change the number of votes required for a proposal execution
+
+```bash
+# Remove relayer
+$ cb-sol-cli admin set-threshold \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --threshold [THRESHOLD]
+```
+:::
+
+## Chain ID
+
+The Chainbridge `chainId` is an arbitrary value used in the bridge for differentiating between the blockchain networks. To not be confused with the chain ID of the network, they are not the same thing. This value needs to be unique (not the same as the chain ID of the network), since two networks might have the same one.
+
+In this example, we set  `99` in `chainId`, because the chain ID of the Mumbai testnet is `80001`, which cannot be represented with a uint8.
+
+## Resource ID
+
+A resource ID is a unique 32-byte value in a cross-chain environment, associated with a certain asset (resource) that is being transferred between networks.
+
+The resource ID is arbitrary, but, as a convention, usually the last byte contains the chain ID of the source chain (the network from which this asset originated from). For example, let’s say we have a certain token that is transferred between two networks, Network1 and Network2. On Network1, the resource ID would be tokenA, and, for Network2, the equivalent would be tokenB (A and B being the last byte containing the ID of the source chain).
+
+## JSON-RPC URL for Polygon PoS
+
+For this guide, we’ll use https://rpc-mumbai.matic.today, a public JSON-RPC URL provided by Polygon, which may have traffic or rate-limits. We advise you to obtain your JSON-RPC URL by an external service like Infura because deploying contracts will send many queries/requests to the JSON-RPC.
```

Comment:
just to clarify,
This URL will be used only to connect with Polygon Mumbai testnet